File: Problems/No.2160_Minimum_Sum_of_Four_Digit_Number_After_Splitting_Digits/minimum_sum_of_four_digits_number_after_splitting_digits.cpp
class Solution {
public:
    int minimumSum(int num) {
        
        // parameter
        std::vector<int> digits;

        // start computing
        for (int i=0; i<4; i++) {

            digits.push_back(num%10);
            num /= 10;

        }
        std::sort(digits.begin(), digits.end());
        
        return ((digits[0]+digits[1])*10 + digits[2] + digits[3]);

    }
};

File: Problems/No.2160_Minimum_Sum_of_Four_Digit_Number_After_Splitting_Digits/minimum_sum_of_four_digits_number_after_splitting_digits.py
class Solution(object):
    def minimumSum(self, num):
        """
        :type num: int
        :rtype: int
        """

        # parameter
        digits = []

        # start computing
        for i in range(4):
            digits.append(num%10)
            num //= 10
        digits.sort()

        return ((digits[0]+digits[1])*10 + digits[2] + digits[3])
